src/data_provider/src/packages/rcpWrapper.h

```diff
@@ -0,0 +1,679 @@
+                }
+            };
+
+            const auto getDataFnc
+            {
+                [this, &filePath](std::istream & data)
+                {
+                    std::string line;
+                    std::string bundleShortVersionString;
+                    std::string bundleVersion;
+
+                    m_location = filePath;
+                    m_source = (filePath.find(UTILITIES_FOLDER) != std::string::npos) ? "utilities" : "applications";
+
+                    while (std::getline(data, line))
+                    {
+                        line = Utils::trim(line, " \t");
+
+                        if (line == "<key>CFBundleName</key>" &&
+                                std::getline(data, line))
+                        {
+                            m_name = getValueFnc(line);
+                        }
+                        else if ((line == "<key>CFBundleShortVersionString</key>" ||
+                                  line == "<key>PackageVersion</key>") &&
+                                 std::getline(data, line))
+                        {
+                            bundleShortVersionString = getValueFnc(line);
+                        }
+                        else if (line == "<key>CFBundleVersion</key>" &&
+                                 std::getline(data, line))
+                        {
+                            bundleVersion = getValueFnc(line);
+                        }
+                        else if (line == "<key>LSApplicationCategoryType</key>" &&
+                                 std::getline(data, line))
+                        {
+                            m_groups = getValueFnc(line);
+                        }
+                        else if (line == "<key>CFBundleIdentifier</key>" &&
+                                 std::getline(data, line))
+                        {
+                            m_description = getValueFnc(line);
+
+                            std::string vendor;
+
+                            if (Utils::findRegexInString(m_description, vendor, bundleIdRegex, 1))
+                            {
+                                m_vendor = vendor;
+                            }
+                        }
+                        else if (line == "<key>InstallPrefixPath</key>" &&
+                                 std::getline(data, line))
+                        {
+                            m_installPrefixPath = getValueFnc(line);
+                        }
+                    }
+
+                    if (!bundleShortVersionString.empty())
+                    {
+                        if (Utils::startsWith(bundleVersion, bundleShortVersionString))
+                        {
+                            m_version = bundleVersion;
+                        }
+                        else
+                        {
+                            m_version = bundleShortVersionString;
+                        }
```

Comment:
This change was included to avoid empty string in version, but I see that now we are not setting the default UNKNOWNVALUE.

Reply:
According to what we talked about in the daily reunion, mandatory keys are "name", "version" and "format".
So, these variables shouldn't have to have a default value. Since they are checked outside the wrapper to be not empty.